api/src/main/java/mcoffin/rogue/EncoderControl.java

```diff
@@ -0,0 +1,4 @@
+public interface EncoderDrive {
+  public double getEncoderDistance();
```

Comment:
I don't understand. It should be getEncoderPosition now. I changed it to
that, and that's what I see. Other than that, I don't see the issue. It
should be returning a double.

On Tue, Oct 20, 2015 at 9:47 PM, Matt Coffin <notifications@github.com>
wrote:

> In api/src/main/java/mcoffin/rogue/EncoderControl.java
> <https://github.com/mcoffin/rogue/pull/18#discussion_r42580972>:
>
> > @@ -0,0 +1,4 @@
> > +public interface EncoderDrive {
> > +  public double getEncoderDistance();
>
> @WillYingling <https://github.com/WillYingling> This still needs a
> re-think.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mcoffin/rogue/pull/18/files#r42580972>.
>